<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataProcessApp</class>
 <widget class="QMainWindow" name="DataProcessApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataProcessApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>601</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Datasets"/>
    <addaction name="actionSelect_Datasets_to_Plot"/>
    <addaction name="actionWrite_Function_for_Plots"/>
    <addaction name="actionChange_Graph_Color"/>
   </widget>
   <addaction name="menuLoad"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Datasets"/>
   <addaction name="actionSelect_Datasets_to_Plot"/>
   <addaction name="actionWrite_Function_for_Plots"/>
   <addaction name="actionChange_Graph_Color"/>
  </widget>
  <action name="actionLoad_Datasets">
   <property name="text">
    <string>Load Datasets</string>
   </property>
   <property name="toolTip">
    <string>Load Dataset</string>
   </property>
  </action>
  <action name="actionSelect_Datasets_to_Plot">
   <property name="text">
    <string>Select Datasets to Plot</string>
   </property>
  </action>
  <action name="actionWrite_Function_for_Plots">
   <property name="text">
    <string>Write Function for Plots</string>
   </property>
  </action>
  <action name="actionChange_Graph_Color">
   <property name="text">
    <string>Change Graph Color</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
